cmd/govim/oswatcher_darwin.go

```diff
@@ -0,0 +1,63 @@
+// +build darwin
+
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/fsnotify/fsevents"
+)
+
+const fRemoved = fsevents.ItemRemoved | fsevents.ItemRenamed
+const fChanged = fsevents.ItemCreated | fsevents.ItemModified | fsevents.ItemChangeOwner
+
+func newOSWatcher(gomodpath string) (*osWatcher, error) {
+	dirpath := filepath.Dir(gomodpath)
+	dev, err := fsevents.DeviceForPath(dirpath)
```

Comment:
I'm totally fine with the restriction you describe in the PR. If people run into problems, let's fix them then